automation/run-integration-tests.sh

```diff
@@ -133,6 +133,8 @@ docker --version && cat /etc/resolv.conf
 
 mkdir -p $EXPORT_DIR
 
+sudo modprobe openvswitch
```

Comment:
STDCI does not have `sudo` so it complains.
@tyll , @cathay4t , any suggestions?

Reply:
I suggest a commend like
```shell
lsmod  | grep -q ^openvswitch || modprobe openvswitch || echo "Please run 'modprobe openvswitch' as root"
```
It will tell the user to load the module if it is not loaded and cannot be loaded.